testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1048,6 +1048,15 @@ public void testQueryLoggingCount()
         });
     }
 
+    /**
+     * Two columns (foo_1 and foo_2_4) must be used in the table definition
```

Comment:
if any data type is fine then maybe reword like:

`The table must have two columns foo_1 and foo_2_4 of any type.`